src/AppRouter.js

```diff
@@ -57,10 +57,16 @@ class AppRouter extends React.Component<Props> {
           screen: DrawerNavigator(
             {
               home: {
-                screen: StackNavigator({ home: { screen: HomeScreen } }),
+                screen: StackNavigator(
+                  { home: { screen: HomeScreen } },
+                  { headerMode: 'none' },
```

Comment:
In theory, this should be inherited from parent stackRouter, but it doesn't work. Idk, maybe it's their bug or I forget to propagate nonReactStatic in some HOC.